api-guides/e-receipts.markdown

```diff
@@ -3,142 +3,179 @@ title: eReceipts Recipe
-      },
-      {
-        "SequenceNumber": 2,
-        "Reference": "RideRefId02",
-        "Description": "Ride Item 02 Description",
-        "Description2": "Ride Item 02 Description2",
-        "Quantity": 1,
-        "Amount": 4.95
-      },
-      {
-        "SequenceNumber": 3,
-        "Reference": "RideRefId03",
-        "Description": "Ride Item 03 Description",
-        "Description2": "Ride Item 03 Description2",
-        "Quantity": 1,
-        "Amount": 10
-      }
-    ]
-  },
-  "CustomFields": [
-    {
-      "Name": "RideCustomField01Name",
-      "Value": "RideCustomField02Value"
-    },
-    {
-      "Name": "RideCustomField02Name",
-      "Value": "RideCustomField02Value"
-    },
-    {
-      "Name": "RideCustomField03Name",
-      "Value": "RideCustomField03Value"
-    }
-  ],
-  "ImageBase64": "<insert Base64 representation of image>"
-}
-```
-
-Test your API in the [API Explorer](/api-explorer/v3-0/Receipts.html).  
-
-
-#### If the user connection is initiated from the Partner site, it should follow Web Flow:
-
-![Web Flow](images/web_flow.JPG)
-
-#### If the connection is initated from the Concur App Center, the user experience should follow App Center Flow:
-
-![App Center Flow](images/Normal-flow.jpg)
+### Getting Started
+
+**New App Center Partners:**  Your Company must have entered into a commercial agreement with Concur.  If you have not yet entered into said agreement with Concur, click [here](https://developer.concur.com/why-concur.html) to contact our business development team
+
+**Existing App**  **Center Partners:**  The Concur Platform team will need to configure your existing sandbox among other things to enable access.  You can make that request by contacting your [Alliance Manager](mailto:AppCenterAlliance@concur.com?subject=Receipts%20v4%20Migration%20Request).
+
+
+### User Connections / Authentication
+
+The below Authentication Methods are available to obtain an access token.  Password Grant and either Authorization or One-Time Password Grants are required for Certification:
+
+- [Password Grant](https://developer.concur.com/api-reference/authentication/apidoc.html#password_grant) – Connect from the Concur App Center.
+- [Authorization Grant](https://developer.concur.com/api-reference/authentication/apidoc.html#auth_grant) – Connect from the Partner App.
+- [One-Time Password Grant](https://developer.concur.com/api-reference/authentication/apidoc.html#otp_grant) – Connect from the Partner App utilizing Email.
+
+
+#### App Center Flow - Connecting from Concur Web / Mobile App Center
+
+1. To begin the authentication flow, a request (temporary) token is obtained via the Concur AppCenter interface.
+2. A logged-in Concur user clicks on the Connect button on the App listing in the App Center and authorizes your app to post data.   App Listing Example below:
+
+![appflow_listing](https://user-images.githubusercontent.com/30883242/33037315-24acf7e4-cdff-11e7-835a-c18c27af9c2d.png)
```

Comment:
Images should now reside in `/assets/img/api-guides/e-receipts`. Could you update the links to point to the right images?

Ex. `![appflow_listing](/assets/img/api-guides/e-receipts/appflow_listing.png)`

Images are in this PR: #1256